internal/binarylog/binarylog_end2end_test.go

```diff
@@ -40,6 +40,8 @@ import (
 	"google.golang.org/grpc/status"
 )
 
+var grpclogLogger = grpclog.Component("binarylog")
```

Comment:
Do we want to do this in a test file?

Reply:
This test is in a separate package, so it either defines it's own logger for the component, or the logger needs to be exported.